projects/ad9656_fmc/zcu102/system_bd.tcl

```diff
@@ -0,0 +1,17 @@
+
+
+## NOTE: With this configuration the #36Kb BRAM utilization is at ~57%
```

Comment:
Comment can be removed as there is no OFFLOAD FIFO necessary for this project

Reply:
Comment removed